test/_util/optionSpies.js

```diff
@@ -1,6 +1,6 @@
 import { options as rawOptions } from 'preact';
 
-/** @type {import('../../src/internal').Options} */
+/** @type {import('preact/src/internal').Options} */
```

Comment:
Technically with the jsconfig.json, this is now possible, but felt out of scope for this PR.